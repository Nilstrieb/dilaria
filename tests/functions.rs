mod common;

run_test!(
    single_call,
    r#"
fn test() {
    print "correct";
}

test();
"#
);

run_test!(
    single_call_expect_return,
    r#"
fn test() {
    print "correct1";
}

test();

print "correct2";
"#
);
