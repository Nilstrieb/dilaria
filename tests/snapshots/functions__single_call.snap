---
source: tests/functions.rs
assertion_line: 3
expression: output

---
"correct\n"
